programmers-coding-test/sangjun/sort/성적이낮은순서로학생출력.js

```diff
@@ -0,0 +1,17 @@
+const obj = {
+    '홍길동': 95,
+    '이순신': 77,
+    '정약용': 90,
+    '장보고': 70
+};
+const name = [];
+const scores = [];
+
+Object.entries(obj).sort(([a1, b1], [a2, b2]) => b1 - b2)
+    .forEach(([a, b]) => {
+        name.push(a);
+        scores.push(b);
+    });
```

Comment:
정렬만 해주고 아래에서 호출할 때 name만 호출하는 방향이 좋음
```suggestion
const students = Object.entries(obj);
students.sort((a, b) => a[1] - b[1]);
```